redhat/frr.spec.in

```diff
@@ -775,6 +775,8 @@ sed -i 's/ -M rpki//' %{_sysconfdir}/frr/daemons
 %changelog
 * Tue Nov  4 2021 Martin Winter <mwinter@opensourcerouting.org> - %{version}
 
+* Tue Mar  1 2022 Donatas Abraitis <donatas.abraitis@gmail.com> - 8.2
```

Comment:
Let is have this Feb 1. Avoid dates in Future.  Please update the top line with thr same date too.

Reply:
Fixed.